README.md

```diff
@@ -8,11 +8,11 @@ The Momento Cache module uses [Momento's PHP SDK](https://docs.momentohq.com/cac
 
```

Comment:
Can you mention that a cache is required as part of the prereqs, and link to the console? actually you already have a console link, but you just mention that an API token is required, not a cache.